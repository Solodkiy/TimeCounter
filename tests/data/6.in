11.40 - 13,30

                  14,18 -   16.34
