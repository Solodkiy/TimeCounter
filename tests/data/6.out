11:40 - 13:30 = 1:50
14:18 - 16:34 = 2:16
                ----
                4:06